blocks/mini-teasers/mini-teasers.js

```diff
@@ -0,0 +1,23 @@
+import { decorateModals } from '../../scripts/scripts.js';
+
+export default function decorate(block) {
+  block.parentNode.prepend(document.createElement('hr'));
+  block.classList.add(...'flex flex-row my-16'.split(' '));
+  [...block.children].forEach((element) => {
+    element.classList.add(...'pr-60 w-full'.split(' '));
+    element.querySelector('img')?.classList.add(...'py-1'.split(' '));
```

Comment:
```suggestion
    element.querySelector('img')?.classList.add('py-1');
```
For one string we don't need an array.

Reply:
Done